pages/help/services-resources/_slug.vue

```diff
@@ -26,38 +26,48 @@
             :text="page.text"
         />
 
-        <divider-way-finder
-            color="help"
-            class="divider-way-finder"
-        />
+        <section-wrapper theme="divider">
+            <divider-way-finder
+                color="help"
+                class="divider-way-finder"
+            />
+        </section-wrapper>
 
-        <h2 class="more-info">
-            More Information
-        </h2>
+        <section-wrapper>
```

Comment:
@dianneweinthal Are we still putting an h2 on the whole FlexibleBlocks section? There's a hidden, screenreader-only one in the FlexibleBlocks component...

@matheusgomes062 depending on what Dianne says we should probably just remove this one

Reply:
(The issue with this is that the title isn't in the same `<section>` as the Flexible Blocks content, so screen readers and such will get confused about what it's referring to.)

Reply:
agreed, let's remove this and let it be handled by the FlexibleBlocks component